baseplate/server/monkey.py

```diff
@@ -10,4 +10,9 @@ def patch_stdlib_queues() -> None:
     import queue
     import gevent.queue
 
-    patch_module(queue, gevent.queue, items=["Queue", "LifoQueue", "PriorityQueue"])
+    monkey.patch_module(queue, gevent.queue, items=["Queue", "LifoQueue", "PriorityQueue"])
+
+
+def is_gevent_patched() -> bool:
+    """Has any gevent patching has occurred?"""
```

Comment:
Please also describe what the return value is for this function.

Reply:
I've added a cast to make sure that it's actually a `bool`.

Do I need to add a return type to the docstring even though I have a `bool` type annotation on the function?